translation-function/plurals.md

```diff
@@ -95,27 +95,29 @@ _Or try_ [_translation-check_](https://github.com/locize/translation-check)_, it
 
 ## Ordinal plurals
 
-There is also support for ordinal numbers _(referring to the ordering or ranking of things, e.g. "1st", "2nd", "3rd" in English)_.
+There is also support for ordinal numbers _(referring to the ordering or ranking of things, e.g. "1st", "2nd", "3rd" in English)_. The `ordinal` option tells the helper to use the ordinal digit
+to determine the plurality key used. E.g., for "32" the ordinal digit is "2" so `key_two` is used.
 
 keys
 
 ```javascript
-// i.e. italian
+// i.e. english
 {
-  "key_one": "singular", // cardinal form
-  "key_other": "other", // cardinal form
-  "key_many": "many" // ordinal form
+  "key_one": "{{position}}st place", // 1st, 21st, 31st
```

Comment:
I think instead position, this is count

Reply:
oof, yep, thank you for the good review.